animareflection/ui
# Add `Alert` Component

## Description

##### Task link: https://trello.com/c/RPi1RFrm/215-alert-component

Added `Alert` recipe, component, and stories (render test is baked in with storybook test runner).

## Test Steps

1) Verify that recipe, component, and story structure is sound.
2) Verify that component renders appropriately in both storybook and downstream apps (including different variants).
